enterprise/server/backends/redis_execution_collector/redis_execution_collector.go

```diff
@@ -276,6 +276,12 @@ func (c *collector) DeleteExecutions(ctx context.Context, iid string) error {
 	return c.rdb.Del(ctx, getExecutionKey(iid)).Err()
 }
 
+// SoftDeleteExecutions decreases the TTL of executions data so they're cleaned up soon,
+// but not immediately.
+func (c *collector) SoftDeleteExecutions(ctx context.Context, iid string) error {
+	return c.rdb.Expire(ctx, getExecutionKey(iid), 5*time.Minute).Err()
```

Comment:
can we put this into the const sections above? also how about make it longer like 15 minutes?

Reply:
I chose 5min because In the last 2 weeks, [the max replication lag was ~2min](https://metrics.buildbuddy.io/d/clickhouse-operator/clickhouse-operator-dashboard?orgId=1&from=2025-06-19T05:12:30.621Z&to=2025-07-07T21:43:27.293Z&timezone=browser&var-exported_namespace=$__all&var-chi=$__all&var-hostname=$__all&refresh=1m&viewPanel=panel-7). The longer the TTL the more memory this data will use. Was thinking we can start at 5min and go up from there if we need